### PR TITLE
calculator app

### PR DESCRIPTION
**Pull request by vignesh235**
Basic Calculator App 
Screenshot from 2022-01-11 08-57-17
Screenshot from 2022-01-11 08-57-39

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,6 @@
-# calculator
+Simple basic calculator app
 
-A new Flutter project.
+Here Simple Screenshots
```
Comment (KaviyaPeriyasamy) [inline README.md]: Simple Screenshots means ? @manikandaprabhu123

File: lib/calc.dart [hunk 2]
```diff
@@ -0,0 +1,127 @@
+import 'package:flutter/material.dart';
```
Comment (KaviyaPeriyasamy) [inline lib/calc.dart]: Rename the file to ```calculator_screen.dart``` @vignesh235

File: lib/calc.dart [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+class clac extends StatefulWidget {
```
Comment (KaviyaPeriyasamy) [inline lib/calc.dart]: Give valid class name @UnstopableSathish

File: lib/calc.dart [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+  const clac({ Key? key }) : super(key: key);
```
Comment (KaviyaPeriyasamy) [inline lib/calc.dart]: Give valid class and state name @UnstopableSathish

File: lib/calc.dart [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+                padding: EdgeInsets.all(10.0),
+                alignment: Alignment.bottomRight,
+                child: Text(text,style: TextStyle(
+                  fontSize: 30.0,fontWeight:FontWeight.w700
+                ),),
+              ),
+            ),
+            Row(
+              children: [
+            customOutlineButton("9"),
```
Comment (KaviyaPeriyasamy) [inline lib/calc.dart]: Avoid code duplication of customOutlineButton and try to generate dynamically @Nandhinidevi123

File: lib/calc.dart [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+           
+          ],
+        ),
+      ),
+    );
+  }
+
+Widget customOutlineButton(String val)
+{
+  // ignore: deprecated_member_use
```
Comment (KaviyaPeriyasamy) [inline lib/calc.dart]: Need clarification on this commented statement @Barathpalanisamy

File: README.md [hunk 7]
```diff
@@ -1,16 +1,6 @@
-# calculator
+Simple basic calculator app
```
Comment (KaviyaPeriyasamy) [inline README.md:2]: Try to give a detailed explanation of the concepts that are used in this project  @manikandaprabhu123

File: lib/calc.dart [hunk 8]
```diff
@@ -0,0 +1,127 @@
...
+    first = 0;
+    second = 0; 
+    
+  }else if(btnText == "+" || btnText=="-" ||btnText== "*" || btnText=="/"){
+    first = int.parse(text);
+    result = "";
+    opp = btnText;
+    
+  }else if(btnText == "=" ){
+      second = int.parse(text);
```
Comment (KaviyaPeriyasamy) [inline lib/calc.dart]: Need clarification on ```parse``` @vignesh235

File: pubspec.lock [hunk 9]
```diff
@@ -43,6 +43,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
```
Comment (KaviyaPeriyasamy) [inline pubspec.lock:44]: Explain in detail about pubspec lock file @Barathpalanisamy

**Review by KaviyaPeriyasamy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (manikandaprabhu123) [reply README.md]: Sorry @KaviyaPeriyasamy  it's spelling mistake.. Actually it is sample screenshots

File: pubspec.lock (hunk 9 above)
Comment (Barathpalanisamy) [reply pubspec.lock:44]: In pubspec.lock it shows the current version of dependency and other identifying information for every immediate and transitive dependency a package relies on.
The lockfile is generated automatically for you by pub when you run pub get, pub upgrade, or pub downgrade.

File: lib/calc.dart (hunk 4 above)
Comment (Sathishkumarvintro) [reply lib/calc.dart]: Class name and state name changed correctly

File: lib/calc.dart (hunk 3 above)
Comment (Sathishkumarvintro) [reply lib/calc.dart]: Valid class name changed

File: lib/calc.dart (hunk 6 above)
Comment (Barathpalanisamy) [reply lib/calc.dart]: When we use this (// ignore: deprecated_member_use) command the outlineButton shows without the Strikethrough mark.But the command is not used it shows the outlinebutton in Strikethrough format. And the sample images is attached 

Screenshot from 2022-01-12 10-22-02
Screenshot from 2022-01-12 10-22-20

File: README.md (hunk 7 above)
Comment (manikandaprabhu123) [reply README.md:2]: **README.md  file updated..**

**Pull request merged**